.github/workflows/python-app.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       matrix:
         # Default builds are on Ubuntu
         os: [ubuntu-latest]
-        python-version: [3.7, 3.8, 3.9, pypy-3.7]
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
I'm curious, why was pypy-3.7 dropped? I don't think that's a bad thing, just curious.

Reply:
I dropped this because I started running into build problems with its dependencies, which I took to be something related to use of the sbol2 package. Now that I've got other issues resolved, I'll try it in one more before the merge and see if it can go forward or not.